src/assets/v1.4.1/compatibility_fallback_handler.json

```diff
@@ -6,6 +6,10 @@
     "canonical": {
       "address": "0xfd0732Dc9E303f09fCEf3a7388Ad10A83459Ec99",
       "codeHash": "0x7c6007a5d711cea8dfd5d91f5940ec29c7f200fe511eb1fc1397b367af3c42f9"
+    },
+    "zksync": {
+      "address": "0x9301E98DD367135f21bdF66f342A249c9D5F9069",
+      "codehash": "0x01000227ab67505fb2fa65c81aceddb0a46ddbf3b974583188beda4c5e90417c"
```

Comment:
https://docs.zksync.io/zksync-era/tooling/foundry/migration-guide/deployment#bytecode-hash

This format doesn't match 1.3.0 zksync code hash. Maybe it was obtained differently? Like an rpc request + hashing?

Reply:
I don't know.

> Maybe it was obtained differently? Like an rpc request + hashing?

Reply:
This is how we generated the previous one:

```
➜ cast keccak $(cast code 0x2f870a80647BbC554F3a0EBD093f11B4d2a7492A --rpc-url https://mainnet.era.zksync.io)
0x017e9a83d5513f503fb85274f4d1ad1811040d7caa31772750ffb08638c28fbb
```
New one:
```
➜ cast keccak $(cast code 0x9301E98DD367135f21bdF66f342A249c9D5F9069 --rpc-url https://mainnet.era.zksync.io)
0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470
```

Reply:
I think we should use the hash generated by the zksolc as the network deployment is specific for non-evm chains. I propose that we update the `codehash` in v1.3.0 contract deployments.

Reply:
There isn't a right or wrong answer here. Updating codehash in v1.3.0 would require a release with breaking changes. Since there's nothing wrong with Shebin's approach, I suggest we continue with that one

Reply:
If you really want to add it, we can explore adding a separate optional field `zksolcCodehash`. That should allow us to add it in a non-breaking manner